<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estatísticas do Brasileirão</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="grafico_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1041</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titulo_grafico">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Aperte o botão para gerar o gráfico desejado</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>911</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_regiao">
       <property name="text">
        <string>Gráfico por Regiões</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QComboBox" name="combo_time2"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="combo_atleta2"/>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="checkBarra">
       <property name="text">
        <string>Gráfico de Barras</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="combo_time1"/>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="btn_comp_time">
       <property name="text">
        <string>Comparar Times</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="combo_atleta1"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Atleta 1:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_atleta">
       <property name="text">
        <string>Gráfico do Atleta</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time 1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_comp_atleta">
       <property name="text">
        <string>Comparar Atletas</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="btn_clube">
       <property name="text">
        <string>Gráfico do Time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="checkPizza">
       <property name="text">
        <string>Gráfico de Pizza</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Atleta 2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time 2:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
